### DISCUSSION TITLE
Per instance OpenSSL params set

### DISCUSSION THREAD

**Discussion by brauliobo** [sparklemotion/mechanize#616] [Q&A]
Hi, I would like to set per Mechanize instance SSL options, specifically the one below:
```
OpenSSL::SSL::SSLContext::DEFAULT_PARAMS[:options] |= OpenSSL::SSL::OP_LEGACY_SERVER_CONNECT
```

Is it possible?

**Comment by flavorjones** [comment]
Thanks for asking this question. Mechanize uses `Net::HTTP::Persistent` to manage HTTPS connections. You may want to take a look at that code, or ask this question upstream. Each agent has its own connection manager which you should be able to access through the attribute `:http` to configure it.
